client/state/selectors/get-recommended-plugins.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns a list of recommended plugins for the given site.
+ *
+ * @param  {Object}                   state   Global state tree
+ * @param  {Number}                   siteId  The ID of the site we're querying
+ * @return {Array<Object>|undefined}          Array of plugin objects.
```

Comment:
Hm, getting `undefined` here is definitely unexpected. Could we instead get `[]` or `null` instead? I can see why we'd want to not use `[]` here, but any reason not to use `null`?

Reply:
I think `null` here should be fine 👍